sdk/src/inflation.rs

```diff
@@ -66,12 +65,7 @@ impl Inflation {
 
     /// portion of total that goes to validators
     pub fn validator(&self, year: f64) -> f64 {
```

Comment:
On clusters with inflation currently enabled (testnet), removing `self.storage(year)` without a rolling update to activate will likely cause a loss of consensus.

I'd:
a. switch `_unused` back to `storage`, and document `storage` as unused
b. Set `DEFAULT_STORAGE` to 0.0, just cause
c. Retain the previous `fn validator()` code, and make `fn storage` private

That'll keep storage rewards active on testnet (who cares), no need for a rolling update, but also clearly marks `storage` as dead weight